pkg/bundle/resources.go

```diff
@@ -135,6 +135,10 @@ func chartURL(location *fleet.HelmOptions, auth Auth) (string, error) {
 		return "", err
 	}
 
+	if resp.StatusCode != 200 {
+		return "", fmt.Errorf("failed to read helm repo from %s, error code: %v, response body: %s", location.Repo+"index.yaml", resp.StatusCode, bytes)
```

Comment:
Are bytes always type `string`? Or can it be a slice of bytes?

Reply:
yeah, doing `%s` with bytes will also work

Reply:
https://golang.org/pkg/fmt/ good catch :+1: I checked the reference and it works with string/slices of bytes